docs/source/models/supported_models.rst

```diff
@@ -414,7 +414,7 @@ Text Generation
     - Models
     - Inputs
     - Example HF Models
-    - :ref:`LoRA <lora>`
+    - :ref:`LoRA (see note) <lora>`
```

Comment:
```suggestion
    - :ref:`LoRA <lora>` (see note)
```

Reply:
Thank you very much for your changes, I'm not very familiar with the document syntax, I'll take some time to study it

Reply:
@DarkLight1337  It seems a bit abrupt ,see: https://vllm--9869.org.readthedocs.build/en/9869/models/supported_models.html

Reply:
Let's remove it from the table then, people can still see the note below.